Don't use backspaces.
Index: html2text/html2text.C
===================================================================
--- html2text.orig/html2text.C	2010-05-23 18:03:58.000000000 +0300
+++ html2text/html2text.C	2010-05-23 18:03:58.000000000 +0300
@@ -187,7 +187,7 @@
   const char *style            = "compact";
   int        width             = 79;
   const char *output_file_name = "-";
-  bool       use_backspaces    = true;
+  bool       use_backspaces    = false;
 
   int i;
   for (i = 1; i < argc && argv[i][0] == '-' && argv[i][1]; i++) {
