Index: html2text/HTMLControl.C
===================================================================
--- html2text.orig/HTMLControl.C	2010-05-23 18:06:55.000000000 +0300
+++ html2text/HTMLControl.C	2010-05-23 18:07:03.000000000 +0300
@@ -372,7 +372,7 @@
             attribute.first = c;
             for (;;) {
               c = get_char();
-              if (!isalpha(c) && c != '-' && c != '_' && c != ':') break;
+              if (!isalnum(c) && c != '-' && c != '_' && c != ':') break;
 	      // Same as in line 352 - Arno
               attribute.first += c;
             }
