diff --git a/src/dhcp6.c b/src/dhcp6.c
index 3c22005c..45851e64 100644
--- a/src/dhcp6.c
+++ b/src/dhcp6.c
@@ -1207,7 +1207,7 @@ dhcp6_sendmessage(struct interface *ifp, void (*callback)(void *))
 	/* We need to ensure we have sufficient scope to unicast the address */
 	/* XXX FIXME: We should check any added addresses we have like from
 	 * a Router Advertisement */
-	if (IN6_IS_ADDR_UNSPECIFIED(&state->unicast) ||
+	if (1 || IN6_IS_ADDR_UNSPECIFIED(&state->unicast) ||
 	    (state->state == DH6S_REQUEST &&
 	    (!IN6_IS_ADDR_LINKLOCAL(&state->unicast) || lla == NULL)))
 	{
